index.html

```diff
@@ -8335,14 +8349,17 @@ <h3 id="Nature"><span class="wip">Nature - Use with Caution</span></h3>
                </td>
                <td>Summon Spirit of Spring
                </td>
-               <td>Summon a spring spirit within 100ft, any creature that is within 15ft of
+               <td>Summon a spring spirit within 100ft, any ally that is within 15ft of
                   the spirit at any point within their turn can take 10ft of free movement on that
-                  turn and within 5ft can take an additional distinct reaction in that round. This
-                  persists until the free movement or additional reaction isn’t used.
+                  turn and within 5ft can take an additional distinct reaction in that round. 
+                  <br><br>
+                  This spirit has Medium+SPI HP and an AC of 12 and persists until the free movement 
+                  or additional reaction isn’t used.  
```

Comment:
Might wanna just put a long rest limit on it? Although actually a long rest states that all abilities end so maybe that's fine? But then we do have other abilities like sanctuary that are allowed - I can't remember the wording of it all but maybe worth checking. I'm also thinking that keeping track of those continuation triggers might actually be kinda tricky - is there a nicer/cleaner way to achieve the same goal or are you not worried about that?

Reply:
The triggers mean it essentially ends the moment people are out of combat so didn't feel the need to specify it there.  Am a lil worried about how it is to track, but I don't think it'll be too bad so am willing to see how it runs.